### ISSUE TITLE
Meaning of emotion labels

### ISSUE THREAD

**Issue by gcunhase** [tlkh/text-emotion-classification#11]
In your [dataset with 5 emotions](https://github.com/tlkh/text-emotion-classification/blob/master/dataset/data.bak.csv), what do each of the labels represent? Is the following correct?

```
0: neutral
1: happy
2: sad
3: hate
4: anger
```

**Comment by surya-ce** [comment]
In your dataset with 5 emotions, what do each of the labels represent? Is the following correct?

**Comment by tlkh** [comment]
Yes, it is correct as far as I recall. However, as the (about 2 year old at this point!) data is not manually labelled, it might be a bit noisy. It is worth double checking if the data is up to your quality standards.
